code/modules/sprite_accessories/_accessory_hair.dm

```diff
@@ -648,7 +648,7 @@
 	name = "Rows Bun"
 	icon_state = "rows3"
 
-/datum/sprite_accessory/hair/sargeant
+/datum/sprite_accessory/hair/sergeant
```

Comment:
```suggestion
/datum/sprite_accessory/hair/sargeant
```

Reply:
We cant fix these do to CDB files

Reply:
The path naming should be fine though? 
I had the icon name kept the old/ wrong way because it’s that name in the .dmi or does the path name be the icon name have to be the same name?